runtime/queries/column_timeseries.go

```diff
@@ -531,5 +531,11 @@ func convertRowsToTimeSeriesValues(rows *drivers.Result, rowLength int, tsAlias
 			}
 		}
 	}
+
+	err := rows.Err()
+	if err != nil {
+		return results, err
```

Comment:
Should `return nil, err` – a function should almost never return both a value and an error